.github/workflows/build-release.yml

```diff
@@ -1,37 +1,34 @@
-name: Build Release
+name: build-release
+on: [workflow_dispatch]
 
-on: [ workflow_dispatch ]
+permissions:
+  contents: write
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    container:
+      image: eclipse-temurin:17-jdk
+      options: --user root
     steps:
+      - run: apt update && apt install git -y && git --version
       - uses: actions/checkout@v2
-      - name: Set up JDK 8
-        uses: actions/setup-java@v1
         with:
-          java-version: 8
-      - name: Cache Iris
-        uses: actions/cache@v2
+          fetch-depth: 0
+      - uses: FabricMC/fabric-action-scripts@v1
+        id : changelog
         with:
-          path: |
-            ~/.gradle/caches
-            ~/.gradle/loom-cache
-            ~/.gradle/wrapper
-            ~/.m2/repository
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
-          restore-keys: |
-            ${{ runner.os }}-gradle-
-      - name: Build Iris jar
-        run: ./gradlew build
-      - name: Extract current branch name
-        shell: bash
-        # bash pattern expansion to grab branch name without slashes
-        run: ref="${GITHUB_REF#refs/heads/}" && echo "::set-output name=branch::${ref////-}"
-        id: ref
-      - name: Upload build artifacts
-        uses: actions/upload-artifact@v2
+          context: changelog
+          workflow_id: build-release.yml
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
         with:
-          name: iris-artifacts-${{ steps.ref.outputs.branch }}
-          # Filter built files to disregard -sources and -dev, and leave only the minecraft-compatible jars.
-          path: build/libs/*[0-9].jar
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
```

Comment:
Previously, we used:

```
key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
```

I assume we still want to use gradle-wrapper.properties as a key in case gradle versions are significant.

Reply:
Ah yikes, I missed adding that back in

Reply:
Solves